### PR TITLE
Changes some config options to do things people have requested.

### PR DESCRIPTION
**Pull request by ghost**
okay so i changed some config options to better reflect some things

obviously council approval will be needed for this to be merged

also i can change any of these on requests.

reasons for-

humans having surnames sort of trashed naming conventions for lizards, plasmamen, podpeople etc
this should fix it and people with single-person names can just be handled by admins

no intercept report -(still obviously disabled on some situations like bleb) is just to reduce powergaming stripsearching security for lol randumb, cap can still go to blue

adds some gamemodes back into rotation and changes chances-specifically traitorabd and traitorabdchan, also changes occurances for some (also adds ragin' magin in with a chance of one-IE it'll almost never happen but kek

gang will end if all antags die

traitors get one more obj-this is just mostly experimental and i can revert that with ease

no filthy nonhuman heads allowed reee

peaceborg disabled for being inferior secborg, admins can just deal with shitler borgs

assistants have maint access

ghosts can spin chairs again (why the fuck was this removed)

people can spawn as shadowpeople, which are not shadowlings and are basically really shitty humans-think hardmode: black people edition

gives summon events back to wiz because i'm a cunt
#### Changelog

:cl:
rscadd: Lizards, phytosians, podpeople, shadowlings, etc can now name themselves properly without having to put dumb spaces in between their name-IE no more walks - the - shores, it can be walks-the-shores or whatever your dumb snowflake name is
rscadd: The round will start on code green and stay that way until it is increased to blue-either by any head of staff or the AI. Relevant gamemodes where this would be retarded are unaffected-IE blob.
rscadd: Some gamemode chances have been increased/decreased, some new ones have been put in rotation.
rscadd: Ghosts can spin chairs again.
rscadd: The wizard federation has reported they have improved their meme machines, and can now summon a variety of horrible events, once again.
rscadd: Nanotrasen has changed it's formerly more progressive stance on nonhuman heads of staff or security following a variety of incidents involving lizard captains. Nanotrasen has not offered an official statement to appease the infuriated non-human clientele.
rscadd: The peacekeeper module, for being hilariously ineffective, has been disabled.
rscadd: Assistants have maint access again.
/:cl:

### CODE REVIEW

**Comment by utahclock** [comment]
i dont think tratadbchan is coded. only tratabd that i know of.

**Comment by oisin100** [comment]
Github config files are not taken into account on the host.

Also, None of the weight values for gamemodes are synchronized with our current settings on the host so your updated values cannot be used due to inaccuracy.

And every PR requires councilmin approval... Thats the point of a headcoder...

**Comment by ghost** [comment]
as in council council

File: config/game_options.txt [hunk 1]
```diff
@@ -74,29 +74,29 @@ ALERT_DELTA Destruction of the station is imminent. All crew are instructed to o
...
 PROBABILITY CULT             10
-PROBABILITY CHANGELING         0
+PROBABILITY CHANGELING         5
 PROBABILITY WIZARD             10
 PROBABILITY BLOB             10
 PROBABILITY METEOR             0
 PROBABILITY EXTENDED         0
 PROBABILITY GANG             10
-PROBABILITY RAGINMAGES         0
+PROBABILITY RAGINMAGES         1 //lel
```
Comment (Super3222) [inline config/game_options.txt]: You comment out things on notepad files with ## mate

File: config/game_options.txt [hunk 2]
```diff
@@ -165,7 +165,7 @@ PROTECT_ROLES_FROM_ANTAGONIST
 #PROTECT_ASSISTANT_FROM_ANTAGONIST
 
 ## If non-human species are barred from joining as a head of staff
-#ENFORCE_HUMAN_AUTHORITY
+ENFORCE_HUMAN_AUTHORITY
```
Comment (Super3222) [inline config/game_options.txt]: Thank you

File: config/game_options.txt [hunk 3]
```diff
@@ -220,7 +220,7 @@ MINIMAL_ACCESS_THRESHOLD 8
 #JOBS_HAVE_MINIMAL_ACCESS
 
 ## Uncomment to give assistants maint access.
-#ASSISTANTS_HAVE_MAINT_ACCESS
+ASSISTANTS_HAVE_MAINT_ACCESS
```
Comment (Super3222) [inline config/game_options.txt:223]: Why though? I think we both know the serious outcomes of this.

File: config/game_options.txt [hunk 4]
```diff
@@ -275,7 +275,7 @@ ROUNDSTART_RACES human
 ROUNDSTART_RACES lizard
 #ROUNDSTART_RACES fly
 ROUNDSTART_RACES plasmaman
-#ROUNDSTART_RACES shadow
+ROUNDSTART_RACES shadow
```
Comment (Super3222) [inline config/game_options.txt:278]: LOL

File: config/game_options.txt (hunk 1 above)
Comment (Super3222) [reply config/game_options.txt]: But I don't think it can be on the same line either

File: config/game_options.txt (hunk 3 above)
Comment (ghost) [reply config/game_options.txt:223]: it works on /tg/

Comment (Super3222) [reply config/game_options.txt:223]: Eh alright.

File: config/game_options.txt (hunk 1 above)
Comment (ghost) [reply config/game_options.txt]: fine

**Comment by killerx09** [comment]
Don't you dare make pure-changeling have the same chances of appearing as Traitor.

**Comment by killerx09** [comment]
Also title your PRs properly, FFS.

**Comment by ShadowDeath6** [comment]
Gotta go to species_types and set shadowpeople's roundstart var to 1. Also I'd recommend blacklisting them from sec and giving them darksight like they had before.

File: config/game_options.txt (hunk 3 above)
Comment (ShadowDeath6) [reply config/game_options.txt:223]: no

**Comment by ShadowDeath6** [comment]
no all access maint please, blob is already fucking boned as is

**Comment by Ephemeralis** [comment]
Closing until Adam fixes his PR title. Also probably can't be done via a PR anyway, so open an issue about it.

**Comment by ghost** [comment]
@Ephermalis

Stop touching my PR's all this does is get merged when council DOES update config to show it was actually fucking changed

File: config/game_options.txt (hunk 3 above)
Comment (Time-Green) [reply config/game_options.txt:223]: This is the only good thing in this PR.
That and spinning ghost chairs

**Pull request closed without merging**